client/jetpack-onboarding/steps/summary-todo-links.jsx

```diff
@@ -0,0 +1,58 @@
+/** @format */
```

Comment:
Having this this component sit in a file under `/steps` would make me assume that it's just one of the steps. Do we want to move it a level up, where the `JetpackOnboardingDisclaimer` is?

Reply:
Tricky. I was considering it, but it seemed to go against the intuitive way that our dependencies flow, which is why I put it into this directory. I was even considering adding a `helpers` or so subfolder. What do you think about that? Any better naming suggestions? ( `helpers` is just awful. `util` maybe?)

Reply:
I've just tried moving the file to a `util` subfolder, but then I get the usual linter warning about the `className` not matching the directory. I could work around that with an eslint-ignore statement, but I wonder if it's worth it. Inclined to leave as-is for now (hoping that the fact there's a `summary` step file and a `summery-todo-links` file indicates that the latter somehow belongs to the former).

LMK if you feel this should be changed anyway!

Reply:
Ah, forgot about the disclaimer. Oh well, if there's prior art, I might as well move it up one level.

Reply:
Gah, then again, that requires me to change the `className` -- which right now is `steps__summary-entry`, and also used by the left column.

Reply:
Might be most elegant to also turn the left column into a component and move up one level. I'll do that in a subsequent PR however.

Reply:
Sounds great, thanks!